setup.py

```diff
@@ -84,6 +84,16 @@
 # This overrides using lib_exts, so it has to happen after lib_exts is fully defined
 build_ext, sdist = create_build_ext(lib_exts, cythonize_aliases)
 
+# Force setuptools to consider that there are ext modules, even if empty.
+# See https://github.com/yt-project/yt/issues/2922 and
+# https://stackoverflow.com/a/62668026/2601223 for the fix.
+class BinaryDistribution(Distribution):
+    """Distribution which always forces a binary package with platform name."""
+
+    def has_ext_modules(foo):
```

Comment:
just a small comment here, I think it'd be preferable to have the actual argument name instead of foo..

Reply:
Indeed!

Reply:
Done ;)